plugins/cilium-cni/cni-install.sh

```diff
@@ -6,12 +6,13 @@ HOST_PREFIX=${HOST_PREFIX:-/host}
 CNI_CONF_NAME=${CNI_CONF_NAME:-10-cilium.conf}
 MTU=${MTU:-1450}
 
-CILIUM_CNI=${CILIUM_CNI:-${HOST_PREFIX}/opt/cni/bin/cilium-cni}
+CILIUM_CNI=${CILIUM_CNI:-${HOST_PREFIX}/opt/cni/bin/}
 CILIUM_CNI_CONF=${CILIUM_CNI_CONF:-${HOST_PREFIX}/etc/cni/net.d/${CNI_CONF_NAME}}
 
-# Install cilium-cni binary tohost
+# Install cilium-cni and loopback binary to host
 echo "Installing $CILIUM_CNI ..."
 cp /opt/cni/bin/cilium-cni ${CILIUM_CNI}
+cp /opt/cni/bin/loopback ${CILIUM_CNI}
```

Comment:
Where does the `loopback` binary come from? We don't package it in the cilium container image right now.